src/components/sales/order-row.js

```diff
@@ -0,0 +1,113 @@
+// @flow
+
+import type { OrderOrderedType, OrderFullItemsType } from 'constants/firebase'
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import * as React from 'react'
+
+import Flex from 'components/common/flex'
+import ItemPhoto from 'components/common/item-photo'
+import STYLE from 'constants/style'
+import ORDER from 'constants/order'
+import orderUtil from 'utils/order'
+
+import style from './order-row.style'
+
+export type PropsType = {|
+  order: OrderOrderedType,
+  orderFullItems: OrderFullItemsType,
+|}
+
+class OrderRow extends React.PureComponent<PropsType> {
+  render = () => {
+    return (
+      <Flex grow css={style.base}>
+        {this.renderPhoto()}
+        {this.renderDetails()}
+      </Flex>
+    )
+  }
+
+  renderPhoto = () => {
+    const { orderFullItems } = this.props
+    return (
+      <ItemPhoto item={orderFullItems && orderFullItems[0]} css={style.photo}>
+        {this.renderPhotoStatusDot()}
+      </ItemPhoto>
+    )
+  }
+
+  renderPhotoStatusDot = () => {
+    const { order } = this.props
+
+    const statusDotColorMap = {
```

Comment:
shall we move it in constants?

Reply:
Ideally yeah, but let's leave it as is for now